pkg/varnishcluster/controller/varnishcluster_grafana_dashboard.go

```diff
@@ -37,6 +37,10 @@ func (r *ReconcileVarnishCluster) reconcileGrafanaDashboard(ctx context.Context,
 		return nil
 	}
 
+	if instance.Spec.Monitoring.GrafanaDashboard.Title == "" {
+		instance.Spec.Monitoring.GrafanaDashboard.Title = fmt.Sprintf("Varnish (%s)", instance.Name)
```

Comment:
actually...is it unique enough? Two varnishclusters in different namespaces can have the same name and be monitored by the same grafana. Like two projects having both varnish and both naming their cluster `http-cache` or just `varnish`.  What do you think? sounds possible or I'm being overcautious?

Reply:
Part of me thinks it would be user error to name two varnishclusters the same thing in different namespaces, but I could foresee a situation where automation creates two clusters with the same name. I don't think that's being overly cautious, maybe we include the namespace in the title?

Reply:
The namespace is guaranteed to be different since we don't allow two varnish clusters in the same namespace.

Reply:
yeah, that's what I was thinking of. Including the namespace sounds good to me. Maybe a bit less pretty, but better be on the safe side here. User can always override it to something else.

Reply:
Something like `Varnish (<namespace>-<cluster_name>)`?

Reply:
maybe do it how kubernetes does it with a slash: `Varnish (<namespace>/<cluster_name>) `?